src/jit/gentree.h

```diff
@@ -479,8 +479,8 @@ struct GenTree
     // happening.
     void CopyCosts(const GenTree* const tree)
     {
-        INDEBUG(gtCostsInitialized =
-                    tree->gtCostsInitialized;) // If the 'tree' costs aren't initialized, we'll hit an assert below.
+        // If the 'tree' costs aren't initialized, we'll hit an assert below.
```

Comment:
Why are all these random comment re-alignments popping up? They seem unrelated to the PR.